.github/actions/setup-eks/action.yml

```
@@ -0,0 +1,29 @@
+name: "Setup EKS"
+description: "Setup an EKS cluster"
+inputs:
+  cluster_name:
+    description: "The name to assign to the EKS cluster"
+    required: false
+    default: "consul-api-gateway-test"
+  region:
+    description: "the aws region to run the conformance test in"
```

Trying to make these less specific to conformance tests since we'll 🤞 be using them in the near future for our e2e tests

```suggestion
    description: "The AWS region to create the cluster in"
```